<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page2</class>
 <widget class="QWizardPage" name="Page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QRadioButton" name="romsRadioButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>316</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>ROMS</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>56</y>
     <width>436</width>
     <height>164</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Input Data Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addFileButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse File(s)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="momRadioButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>316</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>MOM</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>27</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Basis data files:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeFileButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove File(s)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="popRadioButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>316</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>POP</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputVDFtext">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>263</y>
     <width>436</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="vdfLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>237</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output VDF File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseOutputVdfFile">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>232</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
